src/wrappers.jl

```diff
@@ -33,13 +35,23 @@ CommonRLInterface.actions(w::MyActionWrapper) = [-1, 1]
 actions(w) # will now return [-1, 1]
 ```
 """
-abstract type AbstractWrapper end
+abstract type AbstractWrapper <: AbstractEnv end
 
 """
-Indicate what the wrapped environment is for an AbstractWrapper (see the AbstractWrapper docstring)
+Indicate what the wrapped environment is for an AbstractWrapper (see the AbstractWrapper docstring).
 """
 function wrapped_env end
 
+"""
+Returns the recursively unwrapped environment.
```

Comment:
I know what this does because you explained it to me, but this docstring is not that clear. I think it deserves two sentences.

Also, docstrings should be in imperative form https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation, i.e. "Return" instead of "Returns"

Reply:
Fixed "Returns" to "Return". Changed the function name and description for clarity.

In terms of use cases, the two that come to mind are:
1. Unwrapping an environment after training,
2. Wanting to get information (Such as parameters) from the unwrapped environment.

There already exists an interface to get underneath the wrapper, so I think it's worthwhile to provide the recursive interface similar to [gymnasium](https://gymnasium.farama.org/api/wrappers/#gymnasium.Wrapper.unwrapped).